frontend/app-development/layout/version-control/VersionControlHeader.tsx

```diff
@@ -157,22 +160,27 @@ export const VersionControlHeader = (props: IVersionControlHeaderProps) => {
       });
     }
   };
+
   const repoPushMutation = useRepoPushMutation(org, app);
```

Comment:
I am not sure if you mean by this suggestion that we  modify useRepoPushMutation  to make it take 3  param

Reply:
Yes, that is what I mean - see this comment: https://github.com/Altinn/altinn-studio/pull/10262#discussion_r1177757408

When useRepoPushMutation accepts the `setAltinnSpinner` function, it can use that function in the `onSuccess` property.